### DISCUSSION TITLE
What's the process for suggesting a small change to a template?

### DISCUSSION THREAD

**Discussion by hushnecampus** [projectdiscovery/nuclei-templates#9205] [Q&A] [answered]
I want to make a minor change to a template (in its description it uses an initialism without first stating the phrase in full, which I believe is bad practice - everyone here may know what an ACL is (or guess based on context) but not everyone who gets a report necessarily will).

What's the process here?  Do I create a new branch, make the change, and submit a merge request into main?  If so is there a naming convention for branches?  Any docs I should read?

**Comment by olearycrew** [comment, marked as answer]
> create a new branch, make the change, and submit a merge request into main

Yes exactly! You can read the whole details on contributing here: https://github.com/projectdiscovery/nuclei-templates/blob/main/CONTRIBUTING.md
